Code/Legacy/CrySystem/DebugCallStack.cpp

```diff
@@ -52,6 +52,10 @@ static bool IsFloatingPointException(EXCEPTION_POINTERS* pex);
 extern LONG WINAPI CryEngineExceptionFilterWER(struct _EXCEPTION_POINTERS* pExceptionPointers);
 extern LONG WINAPI CryEngineExceptionFilterMiniDump(struct _EXCEPTION_POINTERS* pExceptionPointers, const char* szDumpPath, MINIDUMP_TYPE mdumpValue);
 
+void MarkThisThreadForDebugging(const char* name);
+void UnmarkThisThreadFromDebugging();
+void UpdateFPExceptionsMaskForThreads();
```

Comment:
You shouldn't need to prototype these 2 functions since theyre not called in this file

Reply:
I intent to group them together since they are related.

Reply:
Unless they are called they are going to be dead-stripped.